### PR TITLE
Add release 1.3.1 to website

### PR DESCRIPTION
**Pull request by rmetzger**

### CODE REVIEW

File: _posts/2017-06-23-release-1.3.1.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+```
+
+You can find the binaries on the updated [Downloads page](http://flink.apache.org/downloads.html).
+
+
+                            
+<h3>        Bug
+</h3>
+<ul>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-6320'>FLINK-6320</a>] -         Flakey JobManagerHAJobGraphRecoveryITCase
```
Comment (fhueske) [inline _posts/2017-06-23-release-1.3.1.md]: Shouldn't this also go to `Tests`?

File: _posts/2017-06-23-release-1.3.1.md [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-6652'>FLINK-6652</a>] -         Problem with DelimitedInputFormat
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-6659'>FLINK-6659</a>] -         RocksDBMergeIteratorTest, SavepointITCase leave temporary directories behind
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-6669'>FLINK-6669</a>] -         [Build] Scala style check errror on Windows
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-6685'>FLINK-6685</a>] -         SafetyNetCloseableRegistry is closed prematurely in Task::triggerCheckpointBarrier
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-6744'>FLINK-6744</a>] -         Flaky ExecutionGraphSchedulingTest
```
Comment (fhueske) [inline _posts/2017-06-23-release-1.3.1.md]: Move to `Tests`?

File: _posts/2017-06-23-release-1.3.1.md [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-6883'>FLINK-6883</a>] -         Serializer for collection of Scala case classes are generated with different anonymous class names in 1.3
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-6886'>FLINK-6886</a>] -         Fix Timestamp field can not be selected in event time case when  toDataStream[T], `T` not a `Row` Type.
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-6896'>FLINK-6896</a>] -         Creating a table from a POJO and use table sink to output fail
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-6899'>FLINK-6899</a>] -         Wrong state array size in NestedMapsStateTable
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-6913'>FLINK-6913</a>] -         Instable StatefulJobSavepointMigrationITCase.testRestoreSavepoint
```
Comment (fhueske) [inline _posts/2017-06-23-release-1.3.1.md]: Move to `Tests`?

File: _posts/2017-06-23-release-1.3.1.md (hunk 1 above)
Comment (rmetzger) [inline _posts/2017-06-23-release-1.3.1.md]: Its generated by JIRA, because people were using the wrong issue type.

**Comment by rmetzger** [comment]
Thanks for the review. I've moved the issues to the Tests section.

**Pull request closed without merging**